multiapi-engine/src/main/java/com/sngular/api/generator/plugin/openapi/OpenApiGenerator.java

```diff
@@ -309,32 +283,26 @@ private void writeModelRefSchema(
     final String[] refSplit = basicSchema.get$ref().split("/");
     properties.put(refSplit[refSplit.length - 1], basicSchema);
```

Comment:
There are three uses of _refSplit[refSplit.length - 1]_ , may be a constant could avoid repetition.

Other point that I'm thinking about my issue (and related with this code) is if it has sense (at least in async) to work just with the name or qhith the full path (to distinguish if the reference is a message or a schema)